clinica/pipelines/pet_surface/pet_surface_pipeline.py

```diff
@@ -69,14 +69,14 @@ def build_input_node(self):
                                             mandatory_inputs=True),
```

Comment:
Call to `check_relative_volume_location_in_world_coordinate_system` must be adapted with the new key name `pet_tracer` instead of `pet_type`

Reply:
Done. Normally, there is no more mention or `pet_type` in PETSurface.